change/@fluentui-react-toolbar-0491daa8-2b96-481e-88d2-92fa5bba1a2b.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
patch release is enough to get rid of the `rc` in package.json?

Reply:
yup

Reply:
TIL